apps/auth/db/reset_password.py

```diff
@@ -78,12 +78,12 @@ def create(self, docs, **kwargs):
 
             raise SuperdeskApiError.badRequestError('Either key:password or email must be provided')
 
-    def store_reset_password_token(self, doc, email, days_alive, user_id):
-        token_ttl = app.config['RESET_PASSWORD_TOKEN_TIME_TO_LIVE']
-        now = utcnow()
+    def store_reset_password_token(self, doc, email, days_alive, user_id, now=None):
+        if not now:
```

Comment:
If this parameter is not used in code, only for test, then probably it's better to just use `unittest.mock.patch` in test,
something like
```
now = utcnow()
with patch('apps.auth.db.reset_password.utcnow', return_value=now):
...
'_created': now,
...
```